src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParser.cs

```diff
@@ -9390,56 +9418,70 @@ internal ulong PlpBytesTotalLength(TdsParserStateObject stateObj)
             return stateObj._longlen;
         }
 
-         private bool TryProcessUDTMetaData(SqlMetaDataPriv metaData, TdsParserStateObject stateObj) {
+        private bool TryProcessUDTMetaData(SqlMetaDataPriv metaData, TdsParserStateObject stateObj)
```

Comment:
This is a result of Visual Studio's auto code alignment. Earlier code was out of indent, and VS fixed.
I don't see any reason to go back to wrong indentation just because it is not related with my feature.